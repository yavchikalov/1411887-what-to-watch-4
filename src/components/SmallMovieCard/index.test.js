import React from 'react';
import {SmallMovieCard} from './';
import renderer from 'react-test-renderer';

const movie = {
  name: `Fantastic Beasts`,
  cover: `img/fantastic-beasts-the-crimes-of-grindelwald.jpg`,
  trailer: `https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4`
};

describe(`SmallMovieCard`, () => {
  it(`snapshot SmallMovieCard`, () => {
    const tree = renderer.create(
        <SmallMovieCard
          name={movie.name}
          cover={movie.cover}
          trailer={movie.trailer}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
