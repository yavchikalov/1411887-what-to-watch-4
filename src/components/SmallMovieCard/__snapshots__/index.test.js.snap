// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard snapshot SmallMovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseOver={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Fantastic Beasts"
      height="175"
      src="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      Fantastic Beasts
    </a>
  </h3>
</article>
`;
